smurfs/src/actions/index.js

```diff
@@ -1,3 +1,38 @@
+import axios from 'axios';
+
+export const FETCH_SMURFS = 'FETCH_SMURFS';
+export const SUCCESS = 'SUCCESS';
+export const FAILURE = 'FAILURE';
+export const POST_SMURF = 'POST_SMURF';
+
+const url = 'http://localhost:3333/smurfs';
+
+
+export const fetchSmurfs = () => dispatch => {
+  dispatch({ type: FETCH_SMURFS });
+  axios
+    .get(url)
+    .then(res => {
+      dispatch({ type: SUCCESS, payload: res.data });
+    })
+    .catch(err => {
+      dispatch({ type: FAILURE, payload: err })
+    })
+}
+
+export const addSmurf = smurf => dispatch => {
+  dispatch ({ type: POST_SMURF });
+  axios
+    .post(url, smurf)
+    .then(res => {
+      dispatch({ type: SUCCESS, payload: res.data });
+    })
+    .catch(err => {
+      dispatch({ type: FAILURE, payload: err });
+    })
+}
+
+
```

Comment:
Your code is really clean and easy to follow, great job with your actions creators!